source/_components/media_player.songpal.markdown

```diff
@@ -46,6 +46,11 @@ endpoint:
   description: API endpoint of the device.
   required: true
   type: string
+ poll:
```

Comment:
Hi, this line must not be indent.

Reply:
Indeed, thanks! I'll blame the editor for the mistake ;-)